src/main/java/com/crud/library/status/LendStatus.java

```diff
@@ -0,0 +1,11 @@
+package com.crud.library.status;
+
+public enum LendStatus {
+
+    AVAILABLE,
+    RETURNED,
```

Comment:
dla mnie to bardzo dziwny enum, ze mamy statusy wypozyczen a nie statusy w jakich sa poszczegolne exemplarze. Dla mnie status exemplarza to AVAILABLE, BORROWED, DAMAGED, LOST.